deploy/2-configure-feeds.ts

```diff
@@ -116,15 +132,10 @@ const func: DeployFunction = async function ({ network }: HardhatRuntimeEnvironm
         maxStalePeriod: DEFAULT_STALE_PERIOD,
       }),
     },
-    pyth: {
-      oracles: [pythOracle.address, addr0000, addr0000],
+    binance: {
```

Comment:
We could add `binance`, keeping `pyth`. This way it would be easier to add tokens using the pyth oracle in the future

Reply:
Ok. I have added `AUTO` price using Pyth. Although we don't need this token, it will just be used as reference to how to configure Pyth for a given token.